mso/resource_mso_schema_site.go

```diff
@@ -173,6 +185,7 @@ func resourceMSOSchemaSiteRead(d *schema.ResourceData, m interface{}) error {
 		apiTemplate := models.StripQuotes(tempCont.S("templateName").String())
 
 		if apiSiteId == stateSiteId && apiTemplate == stateTemplate {
+			// d.SetId(fmt.Sprintf("%s/sites/%s-%s", schemaId, stateSiteId, stateTemplate))
```

Comment:
Should the commented out code be removed?

Reply:
actually left the comment there on purpose that this is the id we would actually want, but I did not want to introduce a breaking change into the resource.